controllers/storagecluster/storagecluster_controller.go

```diff
@@ -144,14 +150,63 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
-	builder := ctrl.NewControllerManagedBy(mgr).
+	noobaaIgnoreTimeUpdatePredicate := predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			if e.ObjectOld == nil || e.ObjectNew == nil {
+				return false
+			}
+			oldObj := e.ObjectOld.(*nbv1.NooBaa)
+			newObj := e.ObjectNew.(*nbv1.NooBaa)
+
+			diff := cmp.Diff(
+				oldObj, newObj,
+				cmpopts.IgnoreFields(metav1.ObjectMeta{}, "ManagedFields", "ResourceVersion"),
+				cmpopts.IgnoreFields(conditionsv1.Condition{}, "LastHeartbeatTime", "LastTransitionTime"),
```

Comment:
@iamniting is there a way to specify the exact field path to ignore? for example `.status.LastHeartbeatTime`, Right now this code might ignore fields we don't want to ignore in cases where we will introduce fields with similar names

Reply:
There is no way to ignore the Path, But there are fewer chances of that as we are passing the exact struct where we would like to ignore the field.

Reply:
We can do that found a way to do so.

Reply:
Done